components/compact-call-out/compact-call-out.config.yml

```diff
@@ -0,0 +1,28 @@
+context:
+  title: A Headline With 30 Characters
+  desc: A description of about 150 characters, give or take. That means we usually only have room for one or two sentences. Here is what that looks like.
+  cta: Call to Action
+  cta_link: '#'
+  cta_class: ''
+variants:
+  - name: Branded
+    notes: |
+      Product icons can be displayed using an additional theme class:
+      - `mzp-t-product-firefox`
+      - `mzp-t-product-beta`
+      - `mzp-t-product-developer`
+      - `mzp-t-product-nightly`
+      - `mzp-t-product-focus`
+      - `mzp-t-product-mozilla`
+      - `mzp-t-product-vpn`
+      - `mzp-t-product-pocket`
+    context:
+      class: mzp-t-product-firefox
+      cta: Download Firefox
+      cta_class: mzp-t-product
+  - name: Dark theme
+    notes: A dark theme is available with the class `mzp-t-dark`. This example also omits any description.
```

Comment:
**Nit:**No need to add the word `also` since folks wouldn't necessarily go from the `call-out` component's dark variant to `call-out-compact`'s dark variant.